src/ashley/editor/decorators.py

```diff
@@ -0,0 +1,23 @@
+"""This module contains custom decorators for draftjs_exporter"""
+import logging
+
+from draftjs_exporter.dom import DOM
+
+logger = logging.getLogger("ashley")
```

Comment:
You won't lose anything being more precise?
```suggestion
logger = logging.getLogger("ashley.editor.decorators")
```